Source/Orts.Simulation/Simulation/RollingStocks/SubSystems/Controllers/MSTSBrakeController.cs

```diff
@@ -168,8 +172,6 @@ public override void UpdatePressure(ref float pressureBar, float elapsedClockSec
                 }
             }
 
-            if (pressureBar > MaxPressureBar())
```

Comment:
Sorry another couple of queries.

If these lines are removed, does that allow "overcharge-protected" tokens to exceed the MaxPressureBar?

Reply:
I modified release and quick release positions to ensure that EQ pressure never goes over MaxPressureBar. Pressure only increases above maximum while overcharge is active.

Reply:
Thanks for that.